docs/book/how-to/control-logging/set-logging-format.md

```diff
@@ -0,0 +1,37 @@
+---
+description: How to set the logging format in ZenML.
+---
+
+# Set logging format
+
+If you want to change the default ZenML logging format, you can do so with the following environment variable:
+
+```bash
+export ZENML_LOGGING_FORMAT='%(asctime)s %(message)s'
+```
+
+The logging format must use the `%`-string formatting style. Check out [this page](https://docs.python.org/3/library/logging.html#logrecord-attributes) for all available attributes.
+
+Note that setting this on the [client environment](../pipeline-development/configure-python-environments/README.md#client-environment-or-the-runner-environment) (e.g. your local machine which runs the pipeline) will **not automatically change the log format on remote pipeline runs**. That means setting this variable locally with only effect pipelines that run locally.
+
+If you wish to configure it also for [remote pipeline runs](../../user-guide/production-guide/cloud-orchestration.md), you can set the `ZENML_LOGGING_FORMAT` environment variable in your pipeline runs environment as follows:
+
+```python
+docker_settings = DockerSettings(environment={"ZENML_LOGGING_FORMAT": "%(asctime)s %(message)s"})
```

Comment:
I like including the required imports in code blocks like this. It would help our users to copy it right away, while helping us to maintain it a bit better.

Reply:
I just copied this from a different docs page, I'll add it.